.github/workflows/main.yml

```diff
@@ -0,0 +1,66 @@
+name: Lint code & Run tests
+
+on:
+  push:
+    branches:
+      - master
+      - develop
```

Comment:
Can we write it like this?
```suggestion
      - [master, develop ]
```
However I am not sure :thinking:

Reply:
Yup, it would work like this as well.

Reply:
Done!